tests/PerformanceHarness/performance_test_basic.py

```diff
@@ -193,7 +193,7 @@ def __init__(self, testHelperConfig: TestHelperConfig=TestHelperConfig(), cluste
 
         Utils.Debug = self.testHelperConfig.verbose
         self.errorExit = Utils.errorExit
-        self.emptyBlockGoal = 1
+        self.emptyBlockGoal = 7
```

Comment:
This prevents a disconnect from happening right before the performance run. Nicer to have it stable before the performance run.